pytorch_lightning/loops/optimization/manual_loop.py

```diff
@@ -55,7 +55,6 @@ def from_training_step_output(
             )
 
         if "loss" in extra:
-            # accumulate the loss. If `accumulate_grad_batches == 1`, no effect.
             # we detach manually as it's expected that it will have a `grad_fn`
             extra["loss"] = extra["loss"].detach().div(normalize)
```

Comment:
remove the `div` and input arg too?

Reply:
I guess it's optional, I don't care. someone can use it if they want.

Reply:
what you prefer

Reply:
i'd remove it - generally not a fan of unused and undocumented code